test/integration/simulacron-helper.js

```diff
@@ -0,0 +1,298 @@
+var helper = require('../test-helper');
```

Comment:
Small nit, i think it would be nice to simply call it `simulacron` instead of `simulacron-helper` for brevity.